### DISCUSSION TITLE
Link ends in @

### DISCUSSION THREAD

**Discussion by rayquazza49** [jaykali/maskphish#62] [Q&A] [answered]
I removed the maskphish directory git clone it again ,I did this 5 time but still I am not getting anything good
![Screenshot from 2021-08-08 21-27-23](https://user-images.githubusercontent.com/78814455/128638326-fb08f70e-36a0-46b2-a608-159f6ce476a2.png)

**Comment by jaykali** [comment]
Can you open [https://is.gd](https://is.gd) website? Please check on your browser. Please let me know.

**Reply by rayquazza49** [reply]
no its is not loading

**Reply by jaykali** [reply]
That means your Internet Service Provider is blocking our Services. Please Use a VPN. Thanks.

**Reply by rayquazza49** [reply]
oh okay wait let me check

**Reply by rayquazza49** [reply]
i need vpn only for maskphish na?

**Reply by jaykali** [reply]
Yes

**Reply by rayquazza49** [reply]
can u suggest me some good vpn for linux ubuntu

**Reply by jaykali** [reply]
Try [Proton VPN](https://protonvpn.com/support/linux-vpn-setup/) or OpenVPN

**Reply by rayquazza49** [reply]
thank you now it is working

**Comment by jaykali** [comment, marked as answer]
Glad to help you.
